packages/mds-api-helpers/package.json

```diff
@@ -14,7 +14,7 @@
   ],
   "scripts": {
     "build": "tsc --build tsconfig.build.json",
-    "test": "DOTENV_CONFIG_PATH=../../.env nyc --use-spawn-wrap --lines 40  ts-mocha --project ../../tsconfig.json"
+    "test": "DOTENV_CONFIG_PATH=../../.env jest --runInBand --forceExit "
```

Comment:
Does this need `--forceExit`? Looks like the mocha tests didn't. I think we should try to isolate usage of this only to cases where it's super necessary, as it *usually* indicates there's a nasty dangling promise bug.